src/containers/genePage/genomeFeatureWrapper.js

```diff
@@ -70,39 +67,41 @@ class GenomeFeatureViewer extends Component {
   }
 
 
-  componentDidMount() {
-    this.loadData();
-  }
-
   componentDidUpdate() {
```

Comment:
Hi @nathandunn not necessarily a problem, just a reminder that `componentDidUpdate` isn't called when the component initially mounts. _If_ you need to call `loadGenomeFeature` when the component initially mounts, you will need to call it in `componentDidMount`